src/lib/index.test.js

```diff
@@ -375,6 +375,21 @@ testRule({
   config: [true, { available: 2022 }],
 
   accept: [{ code: ".messages { overscroll-behavior: contain; }" }],
+
+  reject: [
+    {
+      code: stripIndent`label {
+        & input {
+          border: blue 2px dashed;
+        }
+      }`,
+      message: messages.notBaselineSelector("nesting", 2022),
+      line: 2,
+      column: 9,
+      endLine: 2,
+      endColumn: 16,
```

Comment:
`endColumn` doesn't seem to be correct. Please check 🙇

ref: https://github.com/eslint/css/blob/main/tests/rules/use-baseline.test.js#L460-L480